docs/csharp/whats-new/tutorials/static-abstract-interface-methods.md

```diff
@@ -0,0 +1,147 @@
+
+In this tutorial, you'll learn how to:
+
+> [!div class="checklist"]
+>
+> * Define interfaces with static members.
+> * Use experimental interfaces to define classes that implement interfaces with operators defined.
+> * Create generic algorithms that rely on static interface methods.
+
+## Prerequisites
+
+You’ll need to set up your machine to run .NET 6, including the C# 10 compiler. The C# 10 compiler is available starting with [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) or the [.NET 6 SDK](https://dotnet.microsoft.com/download).
+
+> [!IMPORTANT]
+> *static abstract members in interfaces* is a preview feature. You must Add the `<EnablePreviewFeatures>True</EnablePreviewFeatures>` in your project file. See [Preview features](https://aka.ms/dotnet-warnings/preview-features)for more information. You can experiment with this feature, and the experimental libraries that use it. We will use feedback from the preview cycles to improve the feature, including possibly making changes, before its general release.
+
+## Static abstract interface methods
+
+Let's start with an example. The following method returns the midpoint of two `double` numbers:
+
+```csharp
+public static double MidPoint(double left, double right) =?
+    (left + right) / (2.0);
+```
+
+The same logic would work for any numeric type: `int`, `short`, `long`, `float` `decimal`, or any type that represents a number. You need to have a way to use the `+` operator, `/` operator, and define a value for `2`. You can use the `System.INumber` interface in the `System.Runtime.Experimental` NuGet package to write the preceding method as the following generic method:
+
+:::code language="csharp" source="./snippets/staticinterfaces/Utilities.cs" id="MidPoint":::
+
+Any type that implements the `INumber` interface must include a definition for `operator +`, and `operator /`. The denominator is defined by `(T.One + T.One)` to create the value `2` for any numeric type. The readonly properties `Zero` and `One` are also implemented by every type that implements `INumber`. Using that property rather than `2` forces the denominator to be the same type as the two parameters.
+
+You define static abstract members in an interface using familiar syntax: You add the `static` and `abstract` modifiers to any static member that doesn't provide an implementation. The following example defines an `IBoolean<T>` interface that can be applied to any type that overrides `operator true` and `operator false`:
```

Comment:
ad `IBoolean<T>`: where is the code for this?
The included snippet and the following text is about `IGetNext<T>`. Do I miss something here?

Reply:
Good catch. I had used a different scenario in my earlier draft, but it wasn't as clear. I'll update the text.